src/Components/Facility/Investigations/ShowInvestigation.tsx

```diff
@@ -141,27 +145,23 @@ export default function ShowInvestigation(props: any) {
   if (patientLoading || investigationLoading) {
     return <Loading />;
   }
-
   return (
-    <div className="mx-auto max-w-7xl px-4">
-      <PageTitle
-        title={t("investigations")}
-        className="mx-3 md:mx-4"
-        crumbsReplacements={{
-          [facilityId]: { name: patientData?.facility_object?.name },
-          [patientId]: { name: patientData?.name },
-        }}
-        backUrl={`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}`}
-      />
+    <PrintPreview
+      title={t("Investigation Report for {{name}}", {
```

Comment:
Could you add this translation key to `en` locale files https://github.com/ohcnetwork/care_fe/blob/develop/src/Locale/en/Consultation.json#L25 ?

Reply:
sure